woocommerce/woocommerce-android
# Free Trial: Upgrades screen UI

Closes: #8450 

Closes: #8451


### Description

This PR adds UI for "Upgrades" screen.

### Testing instructions

Not needed but you can open the `UpgradesScreen` class and render Compose previews to see UI details of each state.

### Images/gif


<img width="1588" alt="image" src="https://user-images.githubusercontent.com/5845095/225339282-4c920130-1ee9-4f54-ab67-0839088d6ff5.png">
<img width="1576" alt="image" src="https://user-images.githubusercontent.com/5845095/225339338-f0e169ee-8e65-4a6c-a558-f26f463e36b4.png">


https://user-images.githubusercontent.com/5845095/224367290-83505321-0daa-4edc-b8e2-ce238c3491af.mp4
